WordPress/Classes/ViewRelated/Reader/ReaderTopicsCardCell.swift

```diff
@@ -2,36 +2,126 @@ import UIKit
 
 /// A cell that displays topics the user might like
 ///
-class ReaderTopicsCardCell: ReaderTopicsTableCardCell {
-    private let cellIdentifier = "TopicCell"
+class ReaderTopicsCardCell: UITableViewCell, NibLoadable {
+    // Views
+    @IBOutlet weak var containerView: UIView!
+    @IBOutlet weak var headerLabel: UILabel!
+    @IBOutlet weak var collectionView: UICollectionView!
 
-    override func configure(_ data: [ReaderAbstractTopic]) {
-        super.configure(data)
+    private(set) var data: [ReaderAbstractTopic] = [] {
+        didSet {
+            guard oldValue != data else {
+                return
+            }
 
-        headerTitle = Constants.title
+            collectionView.reloadData()
+        }
+    }
+
+    weak var delegate: ReaderTopicsTableCardCellDelegate?
+
+    func configure(_ data: [ReaderAbstractTopic]) {
+        self.data = data
+    }
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+
+        applyStyles()
+
+        // Configure header
+        headerLabel.text = Constants.title
+
+        // Configure collection view
+        collectionView.showsHorizontalScrollIndicator = false
+        collectionView.register(ReaderInterestsCollectionViewCell.defaultNib,
+                                forCellWithReuseIdentifier: Constants.reuseIdentifier)
     }
 
-    override func setupTableView() {
-        super.setupTableView()
+    private func applyStyles() {
+        headerLabel.font = WPStyleGuide.serifFontForTextStyle(.title2)
 
-        tableView.register(UITableViewCell.self, forCellReuseIdentifier: cellIdentifier)
+        containerView.backgroundColor = .listForeground
+        headerLabel.backgroundColor = .listForeground
+        collectionView.backgroundColor = .listForeground
+
+        backgroundColor = .clear
+        contentView.backgroundColor = .clear
     }
 
-    override func cell(forRowAt indexPath: IndexPath, tableView: UITableView, topic: ReaderAbstractTopic?) -> UITableViewCell {
-        guard let tagTopic = topic as? ReaderTagTopic else {
-            return UITableViewCell()
+    private struct Constants {
+        static let title = NSLocalizedString("You might like", comment: "A suggestion of topics the user might like")
+
+        static let reuseIdentifier = ReaderInterestsCollectionViewCell.classNameWithoutNamespaces()
+    }
+}
+
+// MARK: - Collection View: Datasource & Delegate
+extension ReaderTopicsCardCell: UICollectionViewDelegate, UICollectionViewDataSource {
+    func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        guard let cell = collectionView.dequeueReusableCell(withReuseIdentifier: Constants.reuseIdentifier,
+                                                            for: indexPath) as? ReaderInterestsCollectionViewCell else {
+            fatalError("Expected a ReaderInterestsCollectionViewCell for identifier: \(Constants.reuseIdentifier)")
```

Comment:
```suggestion
        guard let cell = collectionView.dequeueReusableCell(withReuseIdentifier: ReaderTopicsCardCell.defaultReuseID,
                                                            for: indexPath) as? ReaderInterestsCollectionViewCell else {
            fatalError("Expected a ReaderInterestsCollectionViewCell for identifier: \(ReaderTopicsCardCell.defaultReuseID)")
```

Reply:
Done in: 616998ecec24851a03b947bb70a219a03766ce49